src/main/java/org/killbill/billing/plugin/adyen/core/KillbillAdyenNotificationHandler.java

```diff
@@ -204,6 +203,32 @@ private void handleNotification(final NotificationRequestItem item) {
         handleNotification(null, item);
     }
 
+    private void handleChargebackNotification(final NotificationRequestItem item, boolean success) {
+        final NotificationItem notification = new NotificationItem(item);
+        final DateTime utcNow = clock.getUTCNow();
+
+        final AdyenResponsesRecord originalRecord = getResponseRecord(item.getOriginalReference());
```

Comment:
Yes, it can happen if two payment systems process payments for the same merchant accounts. During the migration, Adyen would be configured to send notifications to both systems.

I would just record the row and stop the flow, without throwing an exception (to prevent Adyen from retrying).